src/ios/BranchSDK.m

```diff
@@ -251,13 +234,14 @@ - (void)logout:(CDVInvokedUrlCommand*)command
     [branch logoutWithCallback:^(BOOL changed, NSError *error) {
         CDVPluginResult *pluginResult = nil;
         if (!error) {
-            pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:@"Success"];
+            pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsBool:changed];
         } else {
             pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:[error localizedDescription]];
         }
         [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
     }];
-    self.branchUniversalObj = nil;
+    self.branchUniversalObjArray = nil;
+    self.branchUniversalObjArray = [[NSMutableArray alloc] init];
```

Comment:
I'm not super familiar with objective c, but do we need to release self.branchUniversalArray in addition to setting it as nil? I saw something in a tutorial earlier that said when things were initialized with alloc, they should be released or it becomes a memory leak (same thing applies to anywhere else in this file that we have an alloc IIRC). Though please let me know if I'm wrong.